docker-compose.yml

```diff
@@ -7,6 +7,21 @@ services:
     ports:
       - "5432:5432"
 
+  app-frontend:
+    image: node:4.4.7-wheezy
```

Comment:
Turns out slim doesn't have the packages necessary to unzip and install the prebuilt phantomjs dependency. Since this isn't something we deploy I'm less worried about the container size.